client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/site-boost-column/boost-license-info-modal.tsx

```diff
@@ -45,23 +47,28 @@ export default function BoostLicenseInfoModal( { onClose, site, upgradeOnly }: P
 			isA4AEnabled
 				? [
 						'jetpack-agency-dashboard-sites',
-						dataViewsState?.search,
-						dataViewsState?.page,
-						{
-							issueTypes: getSelectedFilters( dataViewsState?.filters ),
-							showOnlyFavorites: showOnlyFavorites || false,
-						},
-						dataViewsState.sort,
-						dataViewsState?.perPage,
-						...( agencyId ? [ agencyId ] : [] ),
+						hashParameters( [
+							dataViewsState?.search,
+							dataViewsState?.page,
+							{
+								issueTypes: getSelectedFilters( dataViewsState?.filters ),
+								showOnlyFavorites: showOnlyFavorites || false,
+								showOnlyDevelopmentSites: showOnlyDevelopmentSites || false,
+							},
+							dataViewsState.sort,
+							dataViewsState?.perPage,
+							...( agencyId ? [ agencyId ] : [] ),
+						] ),
 				  ]
 				: [
 						'jetpack-agency-dashboard-sites',
-						search,
-						currentPage,
-						filter,
-						sort,
-						...( agencyId ? [ agencyId ] : [] ),
+						hashParameters( [
+							search,
+							currentPage,
+							filter,
+							sort,
+							...( agencyId ? [ agencyId ] : [] ),
+						] ),
 				  ],
 		[
 			isA4AEnabled,
```

Comment:
<img width="874" alt="Screenshot 2024-08-30 at 10 34 15 AM" src="https://github.com/user-attachments/assets/4b08a443-9dae-4043-bc46-79cce595cc50">

Can we add the missing dependency here?

Reply:
https://github.com/Automattic/wp-calypso/pull/94023/commits/09668a27edaae31e0007cea9a7d9a8751a97f741